### PR TITLE
[174] redesign property details

### PR DESCRIPTION
**Pull request by xdemocle**

### CODE REVIEW

File: src/modules/land-works/components/land-works-card-history/index.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ const SingleViewLandHistory: React.FC<SingleViewRentHistoryProps> = ({ assetId }
           <ExternalLink href={getEtherscanAddressUrl(text.toLowerCase())} className={'by-text'}>
             <div className="renter-row">
               <p>{ens && ens !== text ? ens : shortedRenter}</p>
-              {isYou && <p className="you">you</p>}
+              {!isYou && <p className="you">You</p>}
```
Comment (failfmi) [inline src/modules/land-works/components/land-works-card-history/index.tsx:91]: I guess `!` was added for testing.

**Review by failfmi — COMMENTED** [review]
.

**Comment by failfmi** [comment]
image
Should the text be centered as well? (View in Metaverse)

**Comment by failfmi** [comment]
image
Should this be scaled based on the coordinates found?

File: src/modules/land-works/components/land-works-card-history/index.tsx (hunk 1 above)
Comment (xdemocle) [reply src/modules/land-works/components/land-works-card-history/index.tsx:91]: yes thanks haha

**Comment by xdemocle** [comment]
> image Should this be scaled based on the coordinates found?

what you mean for coordinates found?
Im showing the coordinates fund in decentraland property's coords

**Comment by xdemocle** [comment]
> 

oh you mean the dimension of the whole black block, I understand

**Comment by xdemocle** [comment]
@failfmi I'll tell you when ready

**Comment by xdemocle** [comment]
@failfmi This PR is ready. Thanks :)

**Comment by xdemocle** [comment]
@failfmi https://mui.com/customization/how-to-customize/

I refactored a component to use the styled approach. I was having issues and I was applying too many !important in the global css... so following the documentation, htey said the best was using Styled components.  [Reusable style overrides](https://mui.com/customization/how-to-customize/#2-reusable-style-overrides)

**Pull request merged**